desktop/app/console-helper.js

```diff
@@ -42,8 +42,8 @@ function tee (...writeFns) {
   return t => writeFns.forEach(w => w(t))
 }
 
-const stdErrWriter = t => process.stderr.write(t)
-const stdOutWriter = t => process.stdout.write(t)
+const stdErrWriter = t => process.platform !== 'win32' && process.stderr.write(t)
+const stdOutWriter = t => process.platform !== 'win32' && process.stdout.write(t)
```

Comment:
maybe better to just do the check first so we don't call this and check every time (also can be optimized out with webpack) either 

```
const stdErrWriter = process.platform !== 'win32' ? t =>  && process.stderr.write(t) : () => {}
```

or change the call to tee()